.gitignore

```diff
@@ -37,6 +37,8 @@ wrokspace.code-workspace
 
 ### Others ###
 /rasa/venv/
+/rasa/models/
+/rasa/results/
```

Comment:
The files are still tracked. Remove cache and do another commit.
This is how cache can be removed. 
git rm -r --cached .